src/handbook/peopleops/hiring.md

```diff
@@ -154,3 +154,55 @@ for the open position, take the job posting offline. Candidates applying for a j
 that's no longer available is a waste of their time and a bad experience with
 FlowForge and hurts our reputation.
 
+## Offboarding
+The purpose of the offboarding section in the handbook is to provide a clear and structured framework for employees at FlowForge to follow during the offboarding process. The offboarding process is managed by the People Ops Manager in close collaboration with the CEO, Reporting Managers, and Deel for Payroll at FlowForge.
+
+When a team member or contractor decides to leave FlowForge, they will receive a detailed email before their final date of employment. The email will include information regarding the impact on benefits coverage, stocks, final pay, and the process for returning any company assets.
+
+### Voluntary Resignation
+
+If a team member decides to leave FlowForge, we request them to provide a 30-day notice period by default, unless specified otherwise in their employment contract on Deel. During this period, we expect the team member to continue working to facilitate a smooth handover and enable us to find a suitable replacement. Please note that we generally expect the team member to work for the entire notice period.
+
+If you are considering resigning from your position at FlowForge, we highly recommend discussing your reasons with your manager or the People Ops manager. We are always open to improving and making changes that benefit our team members. While we understand that sometimes leaving is the best option, we encourage you to explore all possible solutions before making a final decision.
+
+Once you have discussed your concerns with your manager and the People Ops team and determined that resigning from FlowForge is the only option, please notify both your manager and the People Ops team of your intention to resign via email. Afterward, we will commence a discussion to establish the steps required to ensure a seamless handover.
+
+### Involuntary Resignation
+
+When a team member is being let go, it is usually due to either performance issues or a change in the company's needs that no longer justifies their role. If the decision is due to performance issues, we will have already provided feedback and given the team member an opportunity to improve. Unfortunately, if the issues cannot be resolved, termination of employment may be necessary.In most cases, we will ask the team member to stop working immediately. The final payment will be made according to the terms of their employment contract.
+
+### Communicating Departures
+
+When a team member voluntarily departs, we may ask if they would like to share their plans with the team. If the departure is involuntary, we will strive to maintain transparency regarding the reasons for the departure, while still respecting the individual's privacy.
```

Comment:
While I agree, it's a bit vague. I don't think we should tell anyone it's performance related, or anything of the kind.

Reply:
I have made changes to this section.